assets/style/main.scss

```diff
@@ -287,7 +287,9 @@ hr {
     margin: 2px 0;
   }
 }
-
+.assessment{max-width:500px; margin: 0 auto;}
+.wooFoundImage{max-width:500px; margin: 0 auto; display: block;}
+.career-matches{max-width:500px; margin: 0 auto;}
```

Comment:
The programmer in me wants `.career-matches` and `.assessment` to be the same class because they're the same code. Although I don't know what I would name it.